sklearn/tests/test_multiclass.py

```diff
@@ -390,7 +400,7 @@ def test_ovr_single_label_predict_proba():
     assert_almost_equal(Y_proba.sum(axis=1), 1.0)
     # predict assigns a label if the probability that the
     # sample has the label is greater than 0.5.
```

Comment:
This is a multiclass classification problem, so the probability threshold is not fixed to 0.5.

```suggestion
    # sample has the label with the greatest predictive probability.
```